docs/rules/idiomatic/directory-package-mismatch.md

```diff
@@ -0,0 +1,127 @@
+policies they test. This may seem to contradict the
+[test-outside-test-package](https://docs.styra.com/regal/rules/testing/test-outside-test-package) rule, which
+says that any test package should have a `_test` suffix in its package path. However, putting tests next to
+the file they target arguably makes it _easier_ to find, and is a common practice in the OPA community. This
+rule therefore by default ignores the `_test` suffix when determining whether the package path matches the
+directory structure.
+
+This behavior can be changed by setting the `exclude-test-suffix` configuration option to `false`, in which
+case package paths with a `_test` suffix also will be required to reside in a directory with a `_test` suffix.
+Notably, this directory structure is used (and enforced) by [Styra DAS](https://docs.styra.com/das) and projects
+that integrate with it.
+
+Setting the `exclude-test-suffix` option to `false` means the example from above would now look like this:
+
+```shell
+# Directory structure                   # Package path
+.
+├── README.md
+└── bundle
+    └── authorization
+        ├── main.rego                   # authorization
+        └── rbac
+            ├── data.json               # authorization.rbac
+            ├── roles
+            │   └── roles.rego          # authorization.rbac.roles
+            ├── roles_test
+            │   └── roles_test.rego     # authorization.rbac.roles_test
+            ├── users
+            │   ├── customers.rego      # authorization.rbac.users
+            │   └── internal.rego       # authorization.rbac.users
+            └── users_test
+                ├── customers_test.rego # authorization.rbac.users_test
+                └── internal_test.rego  # authorization.rbac.users_test
+```
+
+Whichever way you choose is up to you. Consistency is key!
+
+### Bundles
+
+While directory structure doesn't matter to OPA when parsing _policies_, directories parsed as
+[bundles](https://www.openpolicyagent.org/docs/latest/management-bundles/) will read _data_ (`data.json` or
+`data.yaml`) files and insert the data in the `data` document tree based on the directory structure relative
+to the bundle root. Having policies structured in the same manner provides a uniform experience, and makes it
+easier to understand where both policies and data come from.
+
+### Editor Support
+
+Editors integrating Regal's [language server](https://docs.styra.com/regal/language-server) will automatically display
+suggestions for idiomatic package paths based on the directory structure in which a policy resides. The image below
+demonstrates a new policy being created inside an `authorization/rbac/roles` directory, and the editor
+([via Regal](https://docs.styra.com/regal/language-server#code-completions)) suggesting the package path
+ `authorization.rbac.roles`.
+
+![package path auto-completion](../../assets/rules/pkg_name_completion.png)
```

Comment:
I think this will need to similar to this: https://github.com/StyraInc/docs/blob/main/docs/regal/language-server.md?plain=1#L26-L28

Reply:
Ugh :/ But thanks! Fixed in last push.